src/devices/DCMotor/DCMotorWithStartStop.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.DCMotor
+{
+    /// <summary>
+    /// Direct current (DC) motor with Start/Stop
+    /// </summary>
+    public class DCMotorStartStop : DCMotor
```

Comment:
please add a sample somewhere in the bottom of the sample/README.md, otherwise this API might not be used too much